source/fundamentals/crud/compound-operations.txt

```diff
@@ -82,7 +82,7 @@ The example also uses a ``FindOneAndUpdateOptions`` instance to specify the
 following options:
 
 - Exclude the ``_id`` field from the found document with a projection.
-- Specify an upsert, which inserts the query document if no documents match the query.
+- Specify an upsert, which inserts the document if no documents match the query.
```

Comment:
Here it's unclear which "document" you're referring to since you're referring to it as a query filter.